shared/test/participantStore_test.js

```diff
@@ -0,0 +1,139 @@
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+describe("loop.store.ParticipantStore", function() {
+  "use strict";
+
+  var expect = chai.expect;
+  var sandbox, dispatcher, store, now;
+
+  beforeEach(function() {
+    sandbox = LoopMochaUtils.createSandbox();
+
+    dispatcher = new loop.Dispatcher();
+    sandbox.stub(dispatcher, "dispatch");
+
+    now = Date.now();
+    sandbox.stub(Date, "now", function() {
+      return now;
+    });
+
+    store = new loop.store.ParticipantStore(dispatcher);
+  });
+
+  afterEach(function() {
+    sandbox.restore();
+    LoopMochaUtils.restore();
+  });
+
+  describe("#getInitialStoreState", function() {
+    it("should return an object with a property called participants", function() {
+      let initialState = store.getInitialStoreState();
+
+      expect(initialState.participants).not.eql(null);
+    });
+  });
+
+  describe("updatedParticipant", function() {
+    it("should store the participant", function() {
+      let action = {
+        userId: "fakeID",
+        name: "fakeName"
+      };
```

Comment:
Probably should wrap this in the actual action constructor, so it would have caught the misnamed name -> participantName.